sgxs-tools/src/sgx_detect/tests/mod.rs

```diff
@@ -912,6 +1027,14 @@ impl Tests {
                     "Production mode (Intel whitelisted)" => Test(RunEnclaveProdWl),
                 }),
             }),
+            "EnclavesOS system requirements" => Category(Eor, tests: {
```

Comment:
Please ensure this category is hidden when not running in EOS mode. One way to do this would be to add a `Dependency<...> for Eor` with `CONTROL_VISIBILITY = true`.